Structure_Engine/Query/Geometry.cs

```diff
@@ -60,7 +63,10 @@ public static CompositeGeometry Geometry(this ConcreteSection section)
         [Output("outlines", "The geometry of the GeometricalSection as its outline in the global XY plane.")]
         public static IGeometry Geometry(this IGeometricalSection section)
         {
-            return new CompositeGeometry { Elements = section.SectionProfile.Edges.ToList<IGeometry>() };
+            if (section?.SectionProfile?.Edges == null)
+                return new CompositeGeometry();
```

Comment:
Possibly this could return `null`, to distinguish between empty geometries and `null` ones.

Reply:
Personally, I prefer to return an empty one. It is generally advised to return null only when an empty object is not a valid match to the inputs. In this case, I feel an empty `CompositeGeometry` makes sense.